README.md

```diff
@@ -449,6 +449,8 @@ rules:
 
 ### Ignoring Files Globally
 
+**Note**: Ignoring files globally will also disable language server features for those files too.
```

Comment:
Nit for sure, but I don't think you need the ending "too" after having used "also" before :P

Reply:
I made it a little more explicit: "**Note**: Ignoring files will disable most language server features for those files. Only formatting will remain available. Ignored files won't be used for completions, linting, or definitions in other files."

Reply:
Excellent!